source/whats-new.txt

```diff
@@ -12,15 +12,64 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.3:
+
+What's New in 4.3
+-----------------
+
+New features of the 4.3 Java driver release include:
+
+- MongoDB Versioned API. For more information, see our :doc:`Versioned API guide </fundamentals/versioned-api>`. 
+- Added a ``setWindowFields`` pipeline stage to allow the use of window operators 
+- Added support for setting Netty `io.netty.handler.ssl.SslContext <https://netty.io/4.1/api/io/netty/handler/ssl/SslContext.html>`__
+- Added ``snapshot`` (non-speculative) as a ``readConcern`` level for read commands outside of transactions
+- The ``maxConnecting`` property limiting the number of connections you create in a connection pool
```

Comment:
This a constant that cannot be changed neither via public nor via non-advertised means. I have doubts this change is supposed to be documented. But even if it is, we should only say that we limited the rate of establishing new TCP connections per server socket.